src/controllers/guildRoleHandler.ts

```diff
@@ -68,3 +70,74 @@ export async function removeGuildRoleHandler(request: IRequest, env: env) {
     });
   }
 }
+export async function getGuildRolesHandler(request: IRequest, env: env) {
+  const authHeader = request.headers.get("Authorization");
+
+  if (!authHeader) {
+    return new JSONResponse(response.BAD_SIGNATURE, { status: 401 });
+  }
+  try {
+    await verifyAuthToken(authHeader, env);
+    const roles = await getGuildRoles(env);
+    return new JSONResponse({ roles });
+  } catch (err: any) {
+    const error =
+      err?.message === response.ROLE_FETCH_FAILED
+        ? response.ROLE_FETCH_FAILED
```

Comment:
why are we using err of type any, please avoid use any type
suggestion - 
try {
  throw new Error('Oh no!')
} catch (error: Error) {
  // we'll proceed, but let's report it
  reportError({message: error.message})
}
https://kentcdodds.com/blog/get-a-catch-block-error-message-with-typescript

Reply:
Thank you for your comment. But I did try that. 
![image](https://github.com/Real-Dev-Squad/discord-slash-commands/assets/27444637/2fbdf3f7-dbb5-49c4-a75f-894f891d408d)

But I get the above error. 

And currently, we are not reporting anything as you suggest. So if it's required let's create a new ticket for loggers and change it over there.

Let me know if you think this is an important change for you.

Reply:
so you should remove any why to put type for error then, what is the point of doing that